idd/Energy+.idd.in

```diff
@@ -73923,6 +73923,321 @@ HeatPump:PlantLoop:EIR:Heating,
+       \note which is a cubic curve or a lookup table.
+  N6 , \field Minimum Part Load Ratio
+       \minimum> 0.0
+       \maximum 1.0
+       \default 0.1
+       \note Minimum Part Load Ratio (PLR) in between 0 and 1
+  N7 , \field Maximum Part Load Ratio
+       \minimum> 0.0
+       \maximum 1.0
+       \default 1.0
+       \note Maximum Part Load Ratio (PLR) in between 0 and 1
+  A14, \field Fuel Energy Input Ratio Defrost Adjustment Curve name
+       \type object-list
+       \object-list UnivariateFunctions
+       \note EIRDEFROST - Energy Input Ratio Defrost Adjustment Curve name,
+       \note which is a cubic curve or a lookup table function of Outdoor Air Temperature.
+  A15, \field Defrost Control Type
+       \type choice
+       \key timed
+       \key OnDemand
+       \default timed
+       \note Defrost operation control type: timed or OnDemand
+  N8 , \field Defrost Operation Time Fraction
+       \minimum 0
+       \maximum 1
+       \default 0
+       \note Defrost operation time fraction, which will be used for timed defrost control type.
+  N9 , \field Resistive Defrost Heater Capacity
+       \units W
+       \minimum 0
+       \default 0
+       \note The heating power of electric resistive defrost heater. Only used for timed defrost control type.
+  N10, \field Maximum Outdoor Dry-bulb Temperature for Defrost Operation
+       \units C
+       \minimum 0
+       \maximum 10
+       \default 5
+       \note There will be no defrost operation when outdoor air is above this temperature.
+  A16, \field Cycling Ratio Factor Curve Name
+       \type object-list
+       \object-list UnivariateFunctions
+       \note Cycling Ratio Factor (CRF) Curve Name,
+       \note which is a cubic curve or a lookup table function of Cycling Ratio (defined as = PLR/PLRmin).
+  N11, \field Nominal Auxiliary Electric Power
+       \units W
+       \minimum 0
+       \note Nominal Auxiliary Electric Power in [W]
+  A17, \field Auxiliary Electric Energy Input Ratio Function of Temperature Curve Name
+       \type object-list
+       \object-list BivariateFunctions
+       \note Auxiliary Electric EIRFT - Auxiliary Electric Energy Input Ratio Function of Temperature Curve Name,
+       \note which is a biquadratic curve or a lookup table.
+       \note which accounts for system internal fans, pumps, and electronics
+  A18, \field Auxiliary Electric Energy Input Ratio Function of PLR Curve Name
+       \type object-list
+       \object-list UnivariateFunctions
+       \note Auxiliary Electric EIRFPLR - Auxiliary Electric Energy Input Ratio Function of PLR (Part Load Ratio) Curve Name,
+       \note which is a cubic curve or a lookup table.
+  N12; \field Standby Electric Power
+       \units W
+       \minimum 0
+       \default 0
+       \note Standby Electric Power in [W]
+
+HeatPump:AirToWater:FuelFired:Cooling,
+  \memo The object defines a fuel-fired absorption heat pump based on equation-fit models.
+  \min-fields 10
```

Comment:
min-fields 25 (everything).

Reply:
Done

Reply:
Addressed in previous changes---also this was changed to 26 later since a new field "Nominal COP" was added later.